compiler/header.h

```diff
@@ -238,7 +238,7 @@ enum name_types {
     t_string = 0, t_boolean = 1, t_integer = 2, t_routine = 3, t_external = 4,
     t_grouping = 5
 
-/*  If this list is extended, adjust wvn in generator.c  */
+/*  If this list is extended, adjust write_varname in generator.c  */
```

Comment:
This was the only instance of `wvn`. I think this is the right item.

Reply:
Yes - it used to be called `wvn` in some generators but that got unified a while back.